nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -187,9 +179,7 @@ def apply(
         modified_model_graph = graph
         backend = get_backend(modified_model)
 
-        for first_stage_algorithm in self.first_stage_algorithms:
-            algorithm = first_stage_algorithm.algorithm
-
+        for algorithm in self.first_stage_algorithms:
```

Comment:
Are there dependencies in the order of algorithms? I mean, if I apply `ChannelAlignment` first and then `SmoothQuant`, does something change?

Reply:
I've checked and it looks like the order of this algorithms does not matter